apps/confessions-server/.env.example

```diff
@@ -0,0 +1,19 @@
+# The port on which this server runs. We recommend `3005` for convenient
+# local integration with the rest of the services that run within this
+# repository.
+PORT=3005
+
+# The confessions server needs to know where it itself can be accessed in order
+# to be able to create URLs that point to itself.
+CONFESSIONS_SERVER_URL="http://localhost:3005"
+
+ACCESS_TOKEN_SECRET=
+
+# This was inserted by `prisma init`:
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
```

Comment:
add documentation regarding how this database should be initialized for it to be accessible by the app